particles/Tutorial/Javascript/Demo/Game.arcs

```diff
@@ -0,0 +1,60 @@
+// Tutorial Demo
+// Shows how to use Arcs features to create a tic-tac-toe game.
+import 'TicTacToe.arcs'
+import 'GameSchemas.arcs'
+
+particle Game in './source/Game.js'
+  inout Person humanPlayer
+  inout Person computerPlayer
+  in GameState {gameOver} gameState
+  in Move {Text move} humanMove
+  in Move {Text move} computerMove
+  inout Move move
+  check humanPlayer is private
+  check computerPlayer is private
+  consume root
+    provide playerSlot
+    provide greetingSlot
+    provide cellSlot
+
+particle DisplayHumanGreeting in './source/DisplayGreeting.js'
+  in Person player
+  consume greetingSlot
+
+recipe GameDemoRecipe
+  create #volatile #move as move
```

Comment:
What does #move mean here? This handle has a type `Move`, tag `#move` doesn't add anything.

You could maybe have #human #computer on the other 2 move handles, as they have the same type, so tag adds something, but here I don't see it adding any information.